### PR TITLE
Email

### PR DESCRIPTION
**Pull request by githubRaj**
Addition of export of data(csv) file functionality through email.

### CODE REVIEW

**Comment by Nick9500** [comment]
Why did you close this? It didn't work?

**Comment by githubRaj** [comment]
It seems one of the tests is failing

File: src/main/java/nitezh/ministock/activities/PreferencesActivity.java [hunk 1]
```diff
@@ -790,6 +817,53 @@ private void showHelpPortfolio() {
...
+            fOut.close();
+        }
+        catch (IOException e)
+        {
+            Log.e("Exception", "File write failed: " + e.toString());
+        }
+
+        Uri fileuri = Uri.fromFile(file);
+
+        String[] toAddress = {"raj34@live.ca"};
```
Comment (Nick9500) [inline src/main/java/nitezh/ministock/activities/PreferencesActivity.java]: Don't hardcode your personal email address in the code, please. Everyone who clicks email will send you their portfolio.

**Review by Nick9500 — CHANGES_REQUESTED** [review]
Overall good but make the change before merging.

**Review by Nick9500 — APPROVED** [review]
Much better!

**Pull request merged**